smalltalksrc/Slang/CCodeGenerator.class.st

```diff
@@ -1538,10 +1538,18 @@ CCodeGenerator >> generateCASTAt: tast [
 
 { #category : #'CAST translation' }
 CCodeGenerator >> generateCASTAtPut: tast [
+	" Add parentheses to lvalue to correctly generate C code for complex inlining cases. For example in:
+	
+	 '(pTo + anInteger)[i] = pFrom + anInteger[i]'
+	
+	instead of (without parentheses)
+	
+	'pTo + anInteger[i] = pFrom + anInteger[i]'
```

Comment:
I understand that this is the goal of this PR, but this case is not present in the new tests 😞

Reply:
And here also says `anInteger[i]` which is a weird example.